doc/lua_api.txt

```diff
@@ -3560,15 +3508,48 @@ Call these functions only at load time!
       ignored if the recipe contains output. Erasing is then done independently
       from the crafting method.
 * `minetest.register_ore(ore definition)`
+    * Returns an integer uniquely identifying the registered ore.
 * `minetest.register_biome(biome definition)`
+    * Returns an integer uniquely identifying the registered biome.
+* `minetest.unregister_biome(name)`
+    * Unregisters the biome from the engine, and deletes the entry with key
+      `name` from `minetest.registered_biomes`.
 * `minetest.register_decoration(decoration definition)`
+    * Returns an integer uniquely identifying the registered decoration.
+* `minetest.register_schematic(schematic definition)`
+    * Returns an integer uniquely identifying the registered schematic.
+    * If the schematic is loaded from a file, the `name` field is set to the
+      filename.
+    * If the function is called when loading the mod, and `name` is a relative
+      path, then the current mod path will be prepended to the schematic
+      filename.
 * `minetest.override_item(name, redefinition)`
     * Overrides fields of an item registered with register_node/tool/craftitem.
     * Note: Item must already be defined, (opt)depend on the mod defining it.
     * Example: `minetest.override_item("default:mese", {light_source=LIGHT_MAX})`
 * `minetest.clear_registered_ores()`
+    * Clears all ores currently registered.
 * `minetest.clear_registered_biomes()`
+    * Clears all biomes currently registered.
 * `minetest.clear_registered_decorations()`
+    * Clears all decorations currently registered.
+* `minetest.clear_registered_schematics()`
+    * Clears all schematics currently registered.
+* `minetest.register_chatcommand(cmd, chatcommand definition)`
```

Comment:
I'd prefer to have these functions somehow grouped
Suggestion:
1) `Gameplay registration functions`: chat, privilege, auth
2) `Environment registration functions`: items, biome, craft

Reply:
See latest commit